bootstrap.example.toml

```diff
@@ -748,6 +748,12 @@
 # [target.<tuple>] section.
 #jemalloc = false
 
+# Use mimalloc as the global allocator for the compiler. Rust code will prefer `mimalloc` over
+# `jemalloc` if that is also enabled.
```

Comment:
Could we reject setting both? That feels needlessly confusing to me and probably not hugely needed?

Reply:
We can probably just remove jemalloc if mimalloc v3 solves all regressions over it.

Reply:
Maybe name this field `override-allocators` in case we'd want to swap allocators in the future?